lib/local-cluster/local-cluster.cabal

```
@@ -0,0 +1,78 @@
+cabal-version:      3.0
+name:               local-cluster
+version:            0.1.0.0
+synopsis:           Local cluster of cardano nodes
+-- description:
+homepage:           https://github.com/cardanofoundation/cardano-wallet
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Hal, Cardano Foundation
```

```suggestion
author:             HAL, Cardano Foundation
```

(Style: Capital letters, as it's an abbreviation.)